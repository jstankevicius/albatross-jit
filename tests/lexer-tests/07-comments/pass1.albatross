# this is a comment
these are variable names # but this is a comment
