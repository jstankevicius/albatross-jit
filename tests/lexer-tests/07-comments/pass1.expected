1 2 NAME these
7 2 NAME are
11 2 NAME variable
20 2 NAME names
